src/e2e-test/java/io/cdap/cdap/ui/utils/Commands.java

```diff
@@ -74,15 +74,15 @@ public static void testConnection(String connectionType, String connectionId,
     fillConnectionCreateForm(connectionType, connectionId, projectId, serviceAccountPath);
     ElementHelper.clickOnElement(
             Helper.locateElementByCssSelector(
-                    Helper.getCssSelectorByDataTestId("connection-test-button")), 6000);
+                    Helper.getCssSelectorByDataTestId("connection-test-button")), 5);
```

Comment:
The timeout unit is in seconds not milliseconds

Reply:
ok, got it